### PR TITLE
Join room from url

### PR DESCRIPTION
**Pull request by CaseyMichael**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -60,24 +60,45 @@
...
-      var done = false;
       function onPlayerStateChange(event) {
-        if (event.data == YT.PlayerState.PLAYING && !done) {
-          setTimeout(stopVideo, 6000);
-          done = true;
-        }
+        console.debug("onPlayerStateChange", event);
       }
+
+      function playVideo() {
```
Comment (CaseyMichael) [inline client/public/index.html:60]: this seemed to be the easiest way to hook into the player's functions. just put it all on the window #YOLO

File: client/src/Components/Room.js [hunk 2]
```diff
@@ -1,17 +1,45 @@
...
+import queryString from 'query-string';
+import io from 'socket.io-client';
 
 class Room extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    var qRoomId = Object.assign({}, props.match.params);
```
Comment (CaseyMichael) [inline client/src/Components/Room.js:10]: get roomId from url

File: client/src/Components/Room.js [hunk 3]
```diff
@@ -1,17 +1,45 @@
...
+    var qVideoUrl = Object.assign({}, queryString.parse(props.location.search))
```
Comment (CaseyMichael) [inline client/src/Components/Room.js:11]: get videoUrl from query param

**Review by t-laird — APPROVED** [review]
good enough

**Comment by t-laird** [comment]
please resolve your conflicts

**Pull request merged**